packages/sync/src/modules/Module.php

```diff
@@ -373,4 +396,64 @@ public function get_objects_by_id( $object_type, $ids ) {
 
 		return $objects;
 	}
+
+	/**
+	 * Gets a list of minimum and maximum object ids for each batch based on the given batch size.
+	 *
+	 * @access public
+	 *
+	 * @param int         $batch_size The batch size for objects.
+	 * @param string|bool $where_sql  The sql where clause minus 'WHERE', or false if no where clause is needed.
+	 * @param bool        $distinct   True if we should only look at distinct object ids.
+	 *
+	 * @return array|bool An array of min and max ids for each batch.
+	 */
+	public function get_min_max_object_ids_for_batches( $batch_size, $where_sql = false, $distinct = false ) {
+		global $wpdb;
+
+		if ( ! $this->table_name() ) {
+			return false;
+		}
+
+		$results      = array();
+		$table        = $wpdb->{$this->table_name()};
```

Comment:
I really like the `table_name()` implementation in verifying we use only valid table names are used, but does it make sense to add an additional `isset()` or `! empty()` check just in case someone else is extending the `Module` class and didn't define their own `table_name()` there?

Reply:
already done!